docs/configuration-reference.md

```diff
@@ -2017,6 +2017,9 @@ remote_write:
   # host:port to send traces to
   - endpoint: <string>
 
+    # Custom HTTP headers to be sent along with each remote write request.
```

Comment:
Can you note that "authorization" will always be overridden here?

Reply:
Hm, is it really always or just in presence of basic_auth?

Reply:
Ah, that's right, my bad. Let's note that configuring basic auth will override the value for authorization if set in custom headers, then.